client/components/email-verification/email-unverified-notice.jsx

```diff
@@ -141,46 +151,64 @@ export default class EmailUnverifiedNotice extends React.Component {
 			return this.renderEmailSendSuccess();
 		}
 
-		let noticeText = ( <div>
-			<p>
-				<strong>{
-					i18n.getLocaleSlug() === 'en'
-					? i18n.translate( 'To continue, please confirm your email address' )
-					: i18n.translate( 'Please confirm your email address' )
-				}</strong>
-			</p>
-			<p>
-				{ i18n.getLocaleSlug() === 'en'
-					? i18n.translate(
-						'Click the link in the email we sent to %(email)s.',
-						{ args: { email: user.get().email } } )
-					: i18n.translate(
-						'To post and keep using WordPress.com you need to confirm your email address. ' +
-						'Please click the link in the email we sent at %(email)s.',
-						{ args: { email: user.get().email } } )
-				}
-			</p>
-			<p>
+		const noticeText = this.props.noticeText
+			? this.props.noticeText
+			: (
+				<div>
+					<p>
+						<strong>{
+							i18n.getLocaleSlug() === 'en'
```

Comment:
I don't quite understand this. If the text applies to only english why does it have to be translated?
But why have wording very specific to english?